src/EnergyPlus/DaylightingManager.cc

```diff
@@ -7341,76 +7327,76 @@ void DayltgElecLightingControl(EnergyPlusData &state)
-                    FL = 0.0;
-                } else {
+                Real64 FL = 0.0;
+                if (state.dataDaylightingManager->DaylIllum(IL) < thisDaylightControl.IllumSetPoint(IL)) {
                     FL =
                         (thisDaylightControl.IllumSetPoint(IL) - state.dataDaylightingManager->DaylIllum(IL)) / thisDaylightControl.IllumSetPoint(IL);
                 }
 
                 // BRANCH ON LIGHTING SYSTEM TYPE
-                LSYSTP = thisDaylightControl.LightControlType;
+                auto LSYSTP = thisDaylightControl.LightControlType;
+                Real64 FP = 0.0;
                 if (LSYSTP != DataDaylighting::LtgCtrlType::Stepped) {
                     // Continuously dimmable system with linear power curve
                     // Fractional output power required to meet setpoint
                     FP = 1.0;
                     // LIGHT-CTRL-TYPE = CONTINUOUS (LSYSTP = 1)
-                    if (FL <= thisDaylightControl.MinLightFraction) FP = thisDaylightControl.MinPowerFraction;
+                    if (FL <= thisDaylightControl.MinLightFraction) {
+                        FP = thisDaylightControl.MinPowerFraction;
+                    }
                     // LIGHT-CTRL-TYPE = CONTINUOUS/OFF (LSYSTP = 3)
-                    if (FL <= thisDaylightControl.MinLightFraction && LSYSTP == DataDaylighting::LtgCtrlType::ContinuousOff) FP = 0.0;
-                    if (FL > thisDaylightControl.MinLightFraction && FL < 1.0)
+                    if (FL <= thisDaylightControl.MinLightFraction && LSYSTP == DataDaylighting::LtgCtrlType::ContinuousOff) {
+                        FP = 0.0;
+                    }
+                    if (FL > thisDaylightControl.MinLightFraction && FL < 1.0) {
                         FP = (FL + (1.0 - FL) * thisDaylightControl.MinPowerFraction - thisDaylightControl.MinLightFraction) /
                              (1.0 - thisDaylightControl.MinLightFraction);
+                    }
 
                 } else { // LSYSTP = 2
                     // Stepped system
                     FP = 0.0;
-                    if (state.dataDaylightingManager->DaylIllum(IL) > 0.0 &&
-                        state.dataDaylightingManager->DaylIllum(IL) < thisDaylightControl.IllumSetPoint(IL))
+                    // #9060: Use a tolerance, otherwise at very low (< 1e-12) daylighting conditions, you can get a multiplier > 1.0
+                    if (state.dataDaylightingManager->DaylIllum(IL) < 0.1) {
+                        FP = 1.0;
+                    } else if (state.dataDaylightingManager->DaylIllum(IL) < thisDaylightControl.IllumSetPoint(IL)) {
                         FP = double(int(thisDaylightControl.LightControlSteps * FL) + 1) / double(thisDaylightControl.LightControlSteps);
-
-                    if (state.dataDaylightingManager->DaylIllum(IL) == 0.0) FP = 1.0;
+                    }
```

Comment:
This is the actual fix.

Reply:
Seems like this should be the minimum step fraction?

Reply:
Well, that won't work. I guess a 0.1 illuminance is pretty small so shouldn't cause issues.

Reply:
The mishap actually happens between 1e-13 and 1e-14. But 0.1 lux is really small to begin with.